docs/organizations/integrations/jira-integration.md

```diff
@@ -0,0 +1,35 @@
+# Organization Jira integration for Security and risk management
+
+!!! info "This integration is only available for Jira Cloud"
+
+Integrate Jira with [Security and risk management](../managing-security-and-risk.md) to import your Jira issues and manage them all in one place as security items.
+
+## Installing the Jira integration
+
+!!! note
+    -   Before installing the Jira integration, make sure to tag the Jira issues you wish to import with the label **security** (case-insensitive).
+
+    -   Use a Jira account with admin permissions when installing this integration. This lets Codacy access all issues, since the integration inherits the permissions of the account that installs it.
+
+To install the Jira integration:
+
+1.  Open your organization **Integrations**, page **Jira**, and click **Install Jira** to proceed to Atlassian's website.
```

Comment:
@nicklem here, instead of telling users to install right away, I would first talk about the things users must do for Jira to work properly. so before installing, users should review the tasks they would like to monitor to add the security label, a pro tip would be to add that label as a default to all security issues, and make sure the priorities used on those issues are similar to the ones we use to track severity.

to summarize, I would put a bullet before the instal that you have here, and mention these topics before users click Install

Reply:
OK, I'm moving the prerequisites lower down added visibility ([commit](https://github.com/codacy/docs/pull/1843/commits/a3fc093a48579ed475e82d1e4024af58705e953e)).

On a related note, I'm noticing a difference in how we label the Jira priorities. Are they equivalent or is one of these wrong? Thanks 🙏 

* On the [Integrations](https://app.codacy.com/organizations/gh/troubleshoot-codacy/integrations/jira) page: **Highest**, High, Medium and Low
* On the [SRM Configuration](https://app.codacy.com/organizations/gh/troubleshoot-codacy/security/configuration) page and docs: **Blocker**, High, Medium and Low

Reply:
cc @claudiacodacy @mrfyda

Reply:
ufff!! great catch Nico! It's my fault, i think i used an older version of the mockups...

@joanasteodoro could you help me here by replacing **Highest** -> **Blocker**? The SRM is the most accurate source of truth, I'll update the mockups right away

Reply:
I think it should show Jira's default priorities: https://confluence.atlassian.com/jirakb/translating-the-priority-field-1189801690.html

Reply:
I agree with Rafael, if we use the same language it would be less confusing to do the translation @claudiacodacy

Reply:
👍 [updated](https://github.com/codacy/docs/pull/1843/files#diff-30a735ab496ba1650575435a2195b32444487dbb269bb4b685b2e1ad8c7ebe53R182)